subprojects/base-services/src/test/groovy/org/gradle/util/GradleVersionTest.groovy

```diff
@@ -86,7 +86,8 @@ class GradleVersionTest extends Specification {
         version << [
                 '0.9-20101220110000+1100',
                 '0.9-20101220110000-0800',
-                '1.2-20120501110000'
+                '1.2-20120501110000',
+                '1.2-SNAPSHOT',
```

Comment:
Missing more coverage here of how -SNAPSHOT versions compare to other versions WRT Comparable.